exo-setup/src/service-setup.ls

```diff
@@ -9,6 +9,7 @@ require! {
 }
 
 
+# Runs build command in each of the services
 class ServiceSetup extends EventEmitter
```

Comment:
It must be possible to write my own services that run in a particular Docker base image and get set up by whatever command I provide to Exosphere. Also, if I modify the `package.json` file of the service, it needs to run `yarn install` again inside the Docker image. So this command must be specified somewhere, and we need a `ServiceSetup` tool to run them. 

How does dependency installation work inside Docker images? Does Docker run this automatically when booting up the image?